src/test/java/org/elasticsearch/index/query/BaseQueryTestCase.java

```diff
@@ -57,12 +58,14 @@
 
 import java.io.IOException;
 
+import static org.hamcrest.Matchers.*;
+
 @Ignore
 public abstract class BaseQueryTestCase<QB extends BaseQueryBuilder & Streamable> extends ElasticsearchTestCase {
 
-    private static Injector injector;
-    private static IndexQueryParserService queryParserService;
-    private static Index index;
+    protected static Injector injector;
+    protected static IndexQueryParserService queryParserService;
+    protected static Index index;
```

Comment:
do we need to make everything protected? how about exposing a protected method that creates the QueryParseContext which you need in the additional tests?